app/Stacks.js

```diff
@@ -43,25 +50,37 @@ module.exports = class Stack {
 
     for(var i = 0; i < cards.length; i++){
       this.addCard(cards[i]);
+      $(cards[i]).droppable('disable');
```

Comment:
I understand that this is code ported from the `Bonsai` repository, but this type of code is kludgy and hopefully unnecessary. I would like to refactor these types of bolted on solutions instead of adding them to the current version in `Synectic`.